agent/rpm/pbench-agent.spec.j2

```diff
@@ -9,36 +9,45 @@ Source0:        pbench-agent-%{version}.tar.gz
 Buildarch:      noarch
 
 %if 0%{?rhel} == 7
-Requires:  scl-utils, rh-python38, rh-python38-python-pip, rh-redis5
+Requires:  python3, python3-pip
+# For RHEL boxen we need the python development environment in order for the
+# pip3 installs for some modules to be successful.
+Requires:  gcc python3-devel
```

Comment:
Is this required  only for RHEL7?

Reply:
Yes.  :(

Basically, `python-pidfile` has a dependency on `psutils`.  To `pip3 install` `psutils` you need `gcc` and the Python3 development environment in order to build `psutils`.  Since the Fedoras and RHEL 8 both have a `psutils` RPM, we don't need those there, but for RHEL 7 we do.

Reply:
Sigh...